airtime_mvc/application/services/PodcastEpisodeService.php

```diff
@@ -392,10 +392,18 @@ public function _getImportedPodcastEpisodeArray($podcast, $episodes) {
         $episodesArray = array();
         foreach ($rss->get_items() as $item) {
             /** @var SimplePie_Item $item */
-            // If the enclosure is empty or has not URL, this isn't a podcast episode (there's no audio data)
-            $enclosure = $item->get_enclosure();
-            $url = $enclosure instanceof SimplePie_Enclosure ? $enclosure->get_link() : $enclosure["link"];
-            if (empty($url)) { continue; }
+            // If the enclosure is empty or has not URL, this isn't a podcast episode (there's no audio data)     
+            // Added Checks for audio-only enclosures. JBC    
+            foreach ($item->get_enclosures() as $enclosure) {
+				$audio_podcast_found = false;
+				if (substr( $enclosure->get_type(), 0, 5 ) === 'audio') {
+					$url = $enclosure instanceof SimplePie_Enclosure ? $enclosure->get_link() : $enclosure["link"];
+					$audio_podcast_found = true;
```

Comment:
The original code used to check `$url` for `empty()`. Maybe `$audio_podcast_foud = !empty($url)` and skipping the `break` depending on the results makes more sense here?